lib/active_record/virtual_attributes/virtual_total.rb

```diff
@@ -53,7 +53,7 @@ def virtual_total(name, relation, options = {})
       #    # arel => (SELECT sum("disks"."size") where "hardware"."id" = "disks"."hardware_id")
 
       def virtual_aggregate(name, relation, method_name = :sum, column = nil, options = {})
-        return define_virtual_total(name, relation, options) if method_name == :size
+        return virtual_total(name, relation, options) if method_name == :size
```

Comment:
What is the nature of this bug?  What is the difference between these two lines?

Reply:
there is no such thing as `define_virtual_total` - it is a syntax error
but since no one uses virtual aggregate with size, it has never showed up

so dropping this may also be a valid choice